qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -1,42 +1,65 @@
 import os
 import crypt
 import logging
+import math
+import stat
+from sys import version_info as sys_version_info
+from re import search as re_search
+from time import sleep
 from StringIO import StringIO
 from tasks.cephfs.cephfs_test_case import CephFSTestCase
 from tasks.cephfs.fuse_mount import FuseMount
 from teuthology.exceptions import CommandFailedError
+from teuthology.misc import sudo_write_file
 
 log = logging.getLogger(__name__)
 
+def humansize(nbytes):
+    suffixes = ['B', 'K', 'M', 'G', 'T', 'P']
+    i = 0
+    while nbytes >= 1024 and i < len(suffixes)-1:
+        nbytes /= 1024.
+        i += 1
+    nbytes = math.ceil(nbytes)
+    f = ('%d' % nbytes).rstrip('.')
+    return '%s%s' % (f, suffixes[i])
 
 class TestCephFSShell(CephFSTestCase):
     CLIENTS_REQUIRED = 1
 
-    def _cephfs_shell(self, cmd, opts=None, stdin=None):
-        args = ["cephfs-shell", "-c", self.mount_a.config_path]
```

Comment:
We do want to continue testing the behavior that `cephfs-shell <cmd> ...` still works. Let's create or keep a few tests which exercise that?

Reply:
This method is called by `get_cephfs_shell_cmd_output`. So, isn't this being exercised?

Reply:
No, I mean a test which verifies that `cephfs-shell -c conf <cmd ...>` works.

Reply:
Ok... I've added a test that runs `mkdir dir1` using `run_cephfs_shell_cmd()` and then checks if `mount_a.client_remote.run('cephfs-shell'. 'ls')` gives the correct/expected output. Hopefully, this in line what you wanted: "test which verifies that cephfs-shell -c conf <cmd ...> works."